graph/impl/KokkosGraph_RCB_impl.hpp

```diff
@@ -132,9 +132,12 @@ inline void bisect(const coors_view_type &coors_1d, const value_type &init_min_v
   value_type min_val   = init_min_val;
   value_type max_val   = init_max_val;
   value_type p1_weight, p2_weight;
-  value_type prev_weight_ratio = 0;
-  value_type curr_weight_ratio;
-  while (1) {
+  value_type weight_ratio;
+  int cnt = 0;
+  // For now, limit the number of times finding mid point to ten to make RCB work with Sierra T/F coordinates
+  // TODO: switch to use the median is probably a better solution
+  while (cnt <= 10) {
```

Comment:
Then let us make it a proper for loop that will avoid some confusion
```suggestion
  for (int bisection_step = 0; bisection_step < max_bisection_steps; ++bisection_step) {
```

Reply:
@lucbv Thanks for your suggestion. I have changed it to for loop.